esp/esp/users/migrations/0015_auto_20180709_1328.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
For my understanding-- what is happening with these migration files?

Reply:
The migrations change the way the database is set up. They can add fields, remove fields, or change fields. In this case, this is modifying the potential options for the permission field.

Reply:
Any time you make a modification to the database models, you need to create a migration for it. `fab manage:makemigrations` makes the migration files for you, which you then need to `git add`. To then update the database of your own dev server, you do `fab manage:migrate`.

Reply:
oh cool! thanks